ddtrace/payload.py

```diff
@@ -17,9 +18,9 @@ class Payload:
     """
     __slots__ = ('traces', 'size', 'encoder', 'max_payload_size')
 
-    # Default max payload size of 5mb
-    # DEV: Trace agent limit is 10mb, cutoff at 5mb to ensure we don't hit 10mb
-    DEFAULT_MAX_PAYLOAD_SIZE = 5 * 1000000
+    # Default trace agent limit payload size of 10 MB
```

Comment:
The original comment still stands, we want this limit to be < 10mb.... just... in... case....

Go uses 5mb as it's limit, and 5mb is a ton of traces (usually). Maybe we split the difference and call it 8mb?

Reply:
I'm good with keeping 5 MB — the point of this patch is really to have a better algorithm about sending the traces. I've no clue what the "best" payload size might be.

I'll re-set it to 5 MB while fixing the comment ;)